features/support/paths.rb

```diff
@@ -25,6 +25,9 @@ def path_to(page_name)
     #   when /^(.*)'s profile page$/i
     #     user_profile_path(User.find_by_login($1))
 
+    when /^the URL "(.*)"$/
+      $1
+
```

Comment:
I understand why you did this, but I think it may not be ideal code to hardcode URLs into tests; unsure if you know about it but I don't think it's worth changing right now